app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,28 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+  .govuk-breadcrumbs__list {
+    display: flex;
+    padding: 14px 0;
```

Comment:
From [Mia's document](https://docs.google.com/document/d/1OFG1Ln6VeCR_bc0qmUtRwMJU62Ljw4F3HO6hq2yloXk/edit#heading=h.jlvdn5p7ilvu) it looks like we want the spacing above and below the breadcrumbs to be 15px. 

1) the page title itself has spacing on, so I'm not sure we can control this.
2) the `govuk-breadcrumbs` element already has a 14px margin-top and a 10px margin-bottom. If we want to just make the breadcrumb top and bottom spacing equal, can we the margin-bottom on `govuk-breadcrumbs` instead of adding new padding to `govuk-breadcrumbs__list`?

Reply:
Related question, isn't the document talking about the touch targets not being big enough? Is which case the padding shouldn't be added to the list item, it should be added to the link?